pkg/scheduler/internal/cache/cache.go

```diff
@@ -237,6 +240,18 @@ func (cache *cacheImpl) UpdateSnapshot(nodeSnapshot *Snapshot) error {
 			if (len(existing.PodsWithRequiredAntiAffinity) > 0) != (len(clone.PodsWithRequiredAntiAffinity) > 0) {
 				updateNodesHavePodsWithRequiredAntiAffinity = true
 			}
+			if !updateUsedPVCSet {
+				if len(existing.PVCRefCounts) != len(clone.PVCRefCounts) {
+					updateUsedPVCSet = true
+				} else {
+					for pvcKey := range clone.PVCRefCounts {
+						if _, found := existing.PVCRefCounts[pvcKey]; !found {
+							updateUsedPVCSet = true
+							break
+						}
+					}
```

Comment:
Can you make this part UTed?

Reply:
Sure thing added two UT covering:
- same ref count length, same content
- same ref count length, different content

Reply:
I suppose L249 is still not hit. How about building a case like this:

- a pod with PVC0 lands on node0
- updateSnapshot
- delete the pod
- another pod with PVC1 lands on node0
- updateSnapshot

In this case, we should be able to cover L249.

Reply:
Sure thing
```
Add then Remove pod with PVC
```
test case covers ^ and confirmed the block is fully covered with go test coverage